calculator/src/App.js

```diff
@@ -1,16 +1,66 @@
-import React from "react";
+import React, { Component } from "react";
 import "./App.css";
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Text Stats</h1>
-      <p>
-        Type or paste your text here for some vital statistics on your writing!
-      </p>
-      <textarea></textarea>
-    </div>
-  );
+class App extends Component {
+  state = { textArray: [], sentences: [], numberE: [], commas: [] };
+
+  render() {
+    // const characters = this.state.textArray.length;
+    // const sentences = this.state.sentences.length;
+    // const es = this.state.numberE.length;
+    // const commas = this.state.
+    const handleSubmit = e => {
+      e.preventDefault();
+      console.log(e.target.text.value);
+      let text = e.target.text.value;
```

Comment:
Maybe I missed it, but I don't see any string sanitization to account for capital letters. Your code checks for lower case 'e' but I don't see where it would count capital 'E' for the remaining total.